Jobs.Fetcher.Twitter/Helpers/DbReader.cs

```diff
@@ -18,49 +18,98 @@ namespace Jobs.Fetcher.Twitter.Helpers {
 
     public static class DbReader {
 
-        public static User GetUserByUsername(string username, DataLakeTwitterDataContext dbContext) {
-            return dbContext.Users.FirstOrDefault(u => u.Username == username);
+        public static User GetUserByUsername(string username, DataLakeTwitterDataContext dbContext, Logger logger) {
+            try {
```

Comment:
What is this trying to catch? I think if the username does not exist, it would only return a "null", and not raise an error. Perhaps we could use `.First()`, which does issue an error?